test/setup-ca.sh

```diff
@@ -37,7 +37,7 @@ EOF
 fi
 
 # Generate server and client certificates.
-DEFAULT_CNS="pmem-registry pmem-node-controller"
+DEFAULT_CNS="pmem-registry"
```

Comment:
Do we need that name for anything?

Reply:
> Do we need that name for anything?

Not really, but the driver's `certFile` and `keyFile` arguments are using the pmem-registry prefix as default value.

```
pkg/pmem-csi-driver/main.go:    flag.StringVar(&config.CertFile, "certFile", "pmem-registry.pem", "SSL certificate file to use for authenticating client connections")
pkg/pmem-csi-driver/main.go:    flag.StringVar(&config.KeyFile, "keyFile", "pmem-registry-key.pem", "Private key file associated to certificate")
test/setup-ca-kubernetes.sh:REGISTRY_CERT=$(read_key "$tmpdir/pmem-registry.pem")
test/setup-ca-kubernetes.sh:REGISTRY_KEY=$(read_key "$tmpdir/pmem-registry-key.pem")
```

We could rename this to `pmem-controller`, but I am not sure if it effects version-skew tests. I will give It a try.